agent/node.go

```diff
@@ -194,11 +194,9 @@ func (n *Node) run(ctx context.Context) (err error) {
 	// - Given a valid certificate, spin a renewal go-routine that will ensure that certificates stay
 	// up to date.
 	nodeIDChan := make(chan string, 1)
-	caLoadDone := make(chan struct{})
 	go func() {
 		select {
 		case <-ctx.Done():
-		case <-caLoadDone:
 		case nodeID := <-nodeIDChan:
 			logrus.Debugf("Requesting certificate for NodeID: %v", nodeID)
```

Comment:
Yes. This gets called mid-way through `LoadOrCreateSecurityConfig` between `IssueNodeCertificate` and `NodeCertificateStatus`.

We might want to refactor how this works in the future, but this is essentially a hint to the user to what ID we're currently blocked waiting on.